scripts/mkcloud

```diff
@@ -325,7 +325,12 @@ function onhost_enable_ksm
 
 function setuphost
 {
+    local backup=$SKIPSUPPORTCONFIG
+    export SKIPSUPPORTCONFIG=1
+
     ${mkclouddriver}_do_setuphost
+
+    export SKIPSUPPORTCONFIG=$backup
```

Comment:
could be simplified to
`SKIPSUPPORTCONFIG=1 ${mkclouddriver}_do_setuphost`
or adding `local SKIPSUPPORTCONFIG=1`